### PR TITLE
[BISERVER-15169] Adding debug logging to backup and restore process

### PR DESCRIPTION
**Pull request by rmansoor**

### CODE REVIEW

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 30m 13s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl \
api,assemblies/pentaho-solutions,extensions
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 11 test(s) failed:

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMondrianSchemas</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMondrianSchemas_noCatalogs</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMetadata_noModels</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testPerformExportMondrianSchemas_XmlUnsafeDataSourceInfoSaved</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportUsersAndRoles</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMondrianSchemas_AdditionalParametersSaved</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportDatasources</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMetaStore</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMetadata</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.importexport.ZipExportProcessorTest.testPerformExport_withManifest</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.debug(String)" because the return value of "org.pentaho.platform.plugin.services.importexport.ZipExportProcessor.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.importexport.ZipExportProcessorTest.testPerformExport_withoutManifest</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.debug(String)" because the return value of "org.pentaho.platform.plugin.services.importexport.ZipExportProcessor.getRepositoryExportLogger()" is null
```

</p></details>

  
Tests run: **1678**, Failures: **11**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5811/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 27m 4s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl \
api,assemblies/pentaho-solutions,extensions
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 11 test(s) failed:

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMondrianSchemas</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMondrianSchemas_noCatalogs</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMetadata_noModels</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testPerformExportMondrianSchemas_XmlUnsafeDataSourceInfoSaved</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportUsersAndRoles</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMondrianSchemas_AdditionalParametersSaved</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportDatasources</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMetaStore</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMetadata</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.importexport.ZipExportProcessorTest.testPerformExport_withManifest</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.debug(String)" because the return value of "org.pentaho.platform.plugin.services.importexport.ZipExportProcessor.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.importexport.ZipExportProcessorTest.testPerformExport_withoutManifest</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.debug(String)" because the return value of "org.pentaho.platform.plugin.services.importexport.ZipExportProcessor.getRepositoryExportLogger()" is null
```

</p></details>

  
Tests run: **1678**, Failures: **11**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5811/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 25m 59s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl \
api,assemblies/pentaho-solutions,extensions
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 12 test(s) failed:

<details><summary><b>org.pentaho.platform.plugin.services.importer.PlatformImporterTest.testNoMatchingMime</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMondrianSchemas</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMondrianSchemas_noCatalogs</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMetadata_noModels</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testPerformExportMondrianSchemas_XmlUnsafeDataSourceInfoSaved</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportUsersAndRoles</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMondrianSchemas_AdditionalParametersSaved</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportDatasources</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMetaStore</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMetadata</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.importexport.ZipExportProcessorTest.testPerformExport_withManifest</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.debug(String)" because the return value of "org.pentaho.platform.plugin.services.importexport.ZipExportProcessor.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.importexport.ZipExportProcessorTest.testPerformExport_withoutManifest</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.debug(String)" because the return value of "org.pentaho.platform.plugin.services.importexport.ZipExportProcessor.getRepositoryExportLogger()" is null
```

</p></details>

  
Tests run: **1678**, Failures: **12**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5811/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: extensions/src/main/java/org/pentaho/platform/web/http/api/resources/services/FileService.java [hunk 1]
```diff
@@ -95,19 +98,8 @@
...
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Properties;
-import java.util.Set;
-import java.util.StringTokenizer;
+import java.text.SimpleDateFormat;
+import java.util.*;
```
Comment (peterrinehart) [inline extensions/src/main/java/org/pentaho/platform/web/http/api/resources/services/FileService.java:102]: This needs to be expanded

**Review by peterrinehart — CHANGES_REQUESTED** [review]

**Review by angel-ramoscardona — APPROVED** [review]

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 28m 28s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl \
api,assemblies/pentaho-solutions,extensions
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 12 test(s) failed:

<details><summary><b>org.pentaho.platform.plugin.services.importer.PlatformImporterTest.testNoMatchingMime</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMondrianSchemas</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMondrianSchemas_noCatalogs</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMetadata_noModels</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testPerformExportMondrianSchemas_XmlUnsafeDataSourceInfoSaved</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportUsersAndRoles</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMondrianSchemas_AdditionalParametersSaved</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportDatasources</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMetaStore</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporterTest.testExportMetadata</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.info(String)" because the return value of "org.pentaho.platform.plugin.services.exporter.PentahoPlatformExporter.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.importexport.ZipExportProcessorTest.testPerformExport_withManifest</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.debug(String)" because the return value of "org.pentaho.platform.plugin.services.importexport.ZipExportProcessor.getRepositoryExportLogger()" is null
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.importexport.ZipExportProcessorTest.testPerformExport_withoutManifest</b> (click to expand)</summary><p>

```
Cannot invoke "org.pentaho.platform.api.util.IRepositoryExportLogger.debug(String)" because the return value of "org.pentaho.platform.plugin.services.importexport.ZipExportProcessor.getRepositoryExportLogger()" is null
```

</p></details>

  
Tests run: **1678**, Failures: **12**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5811/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 26m 5s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl \
api,assemblies/pentaho-solutions,extensions
```
---
:ok_hand: All tests passed!
 
Tests run: **1678**, Failures: **0**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5811/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 27m 33s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl \
api,assemblies/pentaho-solutions,extensions
```
---
:ok_hand: All tests passed!
 
Tests run: **1678**, Failures: **0**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5811/6/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by angel-ramoscardona — APPROVED** [review]

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 27m 40s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl \
api,assemblies/pentaho-solutions,extensions
```
---
:ok_hand: All tests passed!
 
Tests run: **1678**, Failures: **0**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5811/7/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 26m 37s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl \
api,assemblies/pentaho-solutions,extensions
```
---
:ok_hand: All tests passed!
 
Tests run: **1678**, Failures: **0**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5811/8/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 25m 34s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl \
api,assemblies/pentaho-solutions,extensions
```
---
:ok_hand: All tests passed!
 
Tests run: **1678**, Failures: **0**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5811/9/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: extensions/src/main/java/org/pentaho/platform/web/http/api/resources/services/FileService.java (hunk 1 above)
Comment (rmansoor) [reply extensions/src/main/java/org/pentaho/platform/web/http/api/resources/services/FileService.java]: It has been resolved

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 31m 18s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl \
api,assemblies/pentaho-solutions,extensions
```
---
:ok_hand: All tests passed!
 
Tests run: **1678**, Failures: **0**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5811/10/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 3m 8s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl extensions
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 29m 32s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl \
api,assemblies/pentaho-solutions,extensions
```
---
:ok_hand: All tests passed!
 
Tests run: **1678**, Failures: **0**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5811/12/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**